src/BPEssentials/Core.cs

```diff
@@ -15,6 +15,7 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Reflection;
+using static BPEssentials.ExtensionMethods.Warns.ExtensionPlayerWarns;
```

Comment:
Static using statements are prone for future issues and code readability. Change if possible.
```suggestion

```